chart/values.yaml

```diff
@@ -594,8 +594,9 @@ loki-stack:
     tolerations: []
     priorityClassName: ""
     config:
-      # -- The Loki address to post logs to
-      lokiAddress: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
+      clients:
+        # -- The Loki address to post logs to
+        - url: http://{{ .Release.Name }}-loki:3100/loki/api/v1/push
```

Comment:
Does this need job changes for upgrades?

Reply:
Yes, they're in https://github.com/openebs/mayastor-extensions/pull/396.